src/integrations/watchers/indexable-author-watcher.php

```diff
@@ -91,4 +91,35 @@ public function build_indexable( $user_id ) {
 			$indexable->save();
 		}
 	}
+
+	/**
+	 * Handles the case in which an author is deleted.
+	 *
+	 * @param int      $user_id  User ID.
+	 * @param int|null $reassign The ID of the user the old author's posts are reassigned to.
+	 *
+	 * @return void
+	 */
+	public function handle_user_delete( $user_id, $reassign ) {
```

Comment:
I was confused by the $reassign name, maybe you could change it to $new_user_id or something similar to express that its a user/author id?

Reply:
I thought this was a bool in to first time I read it to maybe check if you should reassign or not.

Reply:
Yeah, I kept the name used by the WordPress hook, but it would totally make sense to choose a clearer one 🙂